packages/x-charts/src/ChartContainer/index.ts

```diff
@@ -1 +1,16 @@
+import type { ChartApi as ChartApiOriginal, PluginsPerSeriesType } from '../context/ChartApi';
+import type { ChartAnyPluginSignature } from '../internals/plugins/models/plugin';
+import type { AllPluginSignatures } from '../internals/plugins/allPlugins';
+
 export * from './ChartContainer';
+
+/**
+ * @deprecated Use `ChartApi` from `@mui/x-charts/context` instead.
```

Comment:
Thanks for this! Created a new item in the [tasks for v9 checklist](https://github.com/mui/mui-x/issues/17955)